Nodejs/Product/Npm/SPI/NpmController.cs

```diff
@@ -33,8 +33,8 @@ internal class NpmController : AbstractNpmLogSource, INpmController {
         private IGlobalPackages _globalPackage;
         private readonly object _lock = new object();
 
-        private readonly FileSystemWatcher _localWatcher;
-        private readonly FileSystemWatcher _globalWatcher;
+        private FileSystemWatcher _localWatcher;
```

Comment:
I think I can revert these changes now, they were more to help with debugging to leak.